workitem/field_definition_blackbox_test.go

```diff
@@ -36,3 +25,53 @@ func TestListFieldDefMarshalling(t *testing.T) {
 		t.Errorf("field should be %v, but is %v", def, unmarshalled)
 	}
 }
+
+func TestFieldDefinition_Marshalling(t *testing.T) {
```

Comment:
@aslakknutsen I've thought about that but as far as I found out, we don't have any test that explicitly test all the other kinds of fields. Not that this is good but I thought why bother then?